data_steward/cdr_cleaner/cleaning_rules/deid/fitbit_device_id.py

```diff
@@ -0,0 +1,139 @@
+"""
+#! locate little baby/summary
+
+Original Issue: DC-3254 ???
+
+#! locate verbose summary
+"""
+import os
+import logging
+
+from common import PIPELINE_TABLES, DEVICE, JINJA_ENV
+from gcloud.bq import BigQueryClient
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+
+LOGGER = logging.getLogger(__name__)
+
+DEVICE_ID = 'device_id'
+
+DEID_FITBIT_DEVICE_ID = JINJA_ENV.from_string("""
+UPDATE `{{project_id}}.{{dataset_id}}.device` AS d
+SET d.{{device_id}} = sub.research_device_id
+FROM `{{project_id}}.{{pipeline_tables}}.wearables_device_id_masking` AS wdim
+WHERE d.person_id = wdim.person_id
+AND d.{{device_id}} = wdim.{{device_id}}
+AND wearables_type = 'fitbit'
+""")
+
+ISSUE_NUMBERS = ['DC-3254']  #! LOCATE
+
+
+class DeidFitbitDeviceId(BaseCleaningRule):
+
+    #! locate update docstring
+    # """
+    # Ensures each domain mapping table only contains records for domain tables
+    # that exist after the dataset has been fully cleaned.
+    # """
+
+    def __init__(self, project_id, dataset_id, sandbox_dataset_id):
+        """
+        Initialize the class with proper info.
+
+        Set the issue numbers, description and affected datasets.  As other
+        tickets may affect this SQL, append them to the list of Jira Issues.
+        DO NOT REMOVE ORIGINAL JIRA ISSUE NUMBERS!
+        """
+
+        desc = (
+            """Every person_id/device_id pair should be given a unique id """
+            """that will be stable across CDR versions""")
+
+        super().__init__(description=desc,
+                         issue_numbers=ISSUE_NUMBERS,
+                         project_id=project_id,
+                         dataset_id=dataset_id,
+                         sandbox_dataset_id=sandbox_dataset_id,
+                         affected_datasets=[
+                             cdr_consts.REGISTERED_TIER_DEID,
+                             cdr_consts.CONTROLLED_TIER_DEID,
+                         ])
+
+        # sandbox_dataset_id
+        self.client = BigQueryClient(project_id=project_id)
```

Comment:
@MichaelSchmidt82 Remove this line and try running again.  Why is this line here?